tests/test_file.py

```diff
@@ -35,3 +37,19 @@ def test_delete_file(self, m):
         self.assertIsInstance(deleted_file, File)
         self.assertTrue(hasattr(deleted_file, 'display_name'))
         self.assertEqual(deleted_file.display_name, "Bad File.docx")
+
+    # download()
+    def test_download_file(self, m):
+        register_uris({'file': ['file_download']}, m)
+
+        self.file.download('canvasapi_file_download_test.txt')
+        self.assertTrue(isfile('canvasapi_file_download_test.txt'))
+        with open('canvasapi_file_download_test.txt') as downloaded_file:
+            self.assertEqual(downloaded_file.read(), '"file contents are here"')
+        cleanup_file('canvasapi_file_download_test.txt')
```

Comment:
It's technically possible (though unlikely) for this to not run if part of the tests fails. That means the file wouldn't be cleaned up and would be left on the test runner's system. To get around this in other places, we've wrapped stuff in a `try`/`finally`.